sharding/data_modeling.rst

```diff
@@ -137,12 +137,13 @@ We've seen some of the benefits of Citus for single-tenant queries, but it can a
     FROM ads GROUP BY account_id
   ORDER BY total_clicks DESC;
 
-There is even a way to run DML statements on multiple tenants. As long as the update statement references data local to its own tenant it can be applied simultaneously to all tenants with a helper UDF called :code:`master_modify_multiple_shards`. Here is an example of modifying all image urls to use secure connections.
+Citus is also able to seamlessly run DML statements on multiple tenants. As long as the update statement references data local to its own tenant it can be applied simultaneously to all tenants. Here is an example of modifying all image urls to use secure connections.
```

Comment:
> As long as the update statement references data local to its own tenant it can be applied simultaneously to all tenants.

This line is a bit confusing, and took me a while to understand. It's not part of this PR, but I'd modify it to be a bit clearer. I *think* the intent is to say that an Update statement shouldn't be able to select from a different shard/tenant, and then update another one. I suspect we error out in these scenarios anyway, so I am also OK taking this statement out.

Reply:
Oops, just merged before this comment appeared.